src/platforms/common/enriching-events/screenshots.mdx

```diff
@@ -4,6 +4,7 @@ sidebar_order: 106
 description: "Learn more about taking screenshots when an error occurs. Sentry pairs the screenshot with the original event, giving you additional insight into issues."
 supported:
   - android
+  - iOS
```

Comment:
The page isn't showing up for ios in the preview, so changing the casing to match the folder in case that's the problem.
```suggestion
  - ios
```